### DISCUSSION TITLE
Welcome to mathnet-filtering Discussions!

### DISCUSSION THREAD

**Discussion by cdrnet** [mathnet/mathnet-filtering#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dinghuafirst** [comment]
![WrongCode](https://user-images.githubusercontent.com/53216052/127597390-47aba3bc-d51b-40cb-b197-926bb1f9e9c5.png)
Is there an error here(IirCoefficients.Notch)？

**Reply by dinghuafirst** [reply]
wc1 is calculated twice ,but wc2 is not calculated.
